sycl/doc/extensions/SubGroup/SYCL_INTEL_sub_group.asciidoc

```diff
@@ -143,7 +153,7 @@ The kernel descriptors below are added to the +info::kernel_device_specific+ enu
 |+info::kernel_device_specific::compile_sub_group_size+
 |N/A
 |+uint32_t+
-|Returns the required sub-group size specified by the kernel, or 0 (if not specified).
+|Returns the required sub-group size specified by the kernel. Returns 0 if the requested size was `auto`, and returns the device's core sub-group size if the requested size was `core`.
```

Comment:
```suggestion
|Returns the required sub-group size specified by the kernel. Returns 0 if the requested size was `auto` (or if it wasn't specified), and returns the device's core sub-group size if the requested size was `core`.
```

Reply:
I think this could be misleading.  With the new wording, a requested size is always specified -- it's just that if no attribute is applied to the kernel, that size is `auto`.

Reply:
```suggestion
|Returns the required sub-group size of the kernel, set implicitly by the implementation or explicitly using a kernel attribute . Returns 0 if the requested size is `auto`, and returns the device's core sub-group size if the requested size is `core`.
```

Reply:
Tightened the description of this query in cebc13f9eb35ffe116910d7e8e8c9ed3c7eb782c.